src/app/itn_orchestrator/schema.graphql

```diff
@@ -96,4 +96,9 @@ input ZkappCommandsDetails {
   feePayers: [PrivateKey!]!
 
   noPrecondition: Boolean!
+
+  """
+  Avoid recently used accounts when generating new zkapp commands
+  """
+  recentlyUsedAccounts: Int!
```

Comment:
this is the size of the queue, right?

would a better name be `accountQueueSize`  or `accountQueueBound`?